client/extensions/woocommerce/lib/countries/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import CA from './CA';
+import US from './US';
+
+// Note: We are not using the other state name resources in calypso
```

Comment:
LOL @justinshreve on further reflection I can't use the locations from the merchant's host - at this point (pre-setup) WooCommerce might not even be installed. Leaving as is.